build.gradle

```diff
@@ -11,12 +11,16 @@ buildscript {
         classpath "com.bertramlabs.plugins:asset-pipeline-gradle:3.2.4"
     }
 }
+//
+//plugins {
+//    id "com.gorylenko.gradle-git-properties" version "2.3.1"
+//}
 
 plugins {
-    id "com.gorylenko.gradle-git-properties" version "2.3.1"
+    id "com.gorylenko.gradle-git-properties" version "2.4.0-rc1"
```

Comment:
Is a release candidate allowed for prod? Would prefer and stable release if there is a version... if not then that's fine to leave as `-rc1`.

Reply:
I'll revert it back, had to do it to start the app locally.

Reply:
Looks like 2.3.1 is last stable (June 2021) - is it not Grails 4 compatible?